gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonCallContext.java

```diff
@@ -59,7 +59,7 @@
  * copies of the object, but with one field changed. The immutability and thread safety of the
  * arguments solely depends on the arguments themselves.
  */
-@BetaApi
+@Beta
 @InternalExtensionOnly
```

Comment:
This class can't be extended so remove the `@InternalExtensionOnly` annotation

Reply:
Done